### ISSUE TITLE
Improve wording for verify this login

### ISSUE THREAD

**Issue by frlan** [vector-im/element-web#17319]
#### Is your suggestion related to a problem? Please describe.

Running a Matrix-server I got some requests from new users that were confused about what this message actually means and what they are supposed to do now:

![Screenshot 2021-05-17 at 10 39 51](https://user-images.githubusercontent.com/1010248/118462414-f1d2bb00-b6fe-11eb-9ee8-84b6bd5dbf14.png)

Therefore I think the text might not good-enought-self-explaining. 

#### Describe the solution you'd like.

I think we should add a more --verbose wording here -- maybe adding a link to a FAQ.

#### Describe alternatives you've considered.
--

#### Additional context
--

**Comment by SimonBrandner** [comment]
Agreed, it's _super_ confusing to average users... I made [this](https://github.com/matrix-org/matrix-react-sdk/pull/5967) to improve the situation a bit and I hope to continue once design gets to it
